radicle-cli/tests/framework/mod.rs

```diff
@@ -162,7 +164,14 @@ impl TestFormula {
                 } else {
                     PathBuf::from(&assertion.program)
                 };
+                log::debug!(target: "test", "Running `{}` in `{}`..", program.display(), self.cwd.display());
 
+                if !program.exists() {
+                    log::error!(target: "test", "Program {} does not exist..", program.display());
```

Comment:
Nitpick, would 'command' be better than 'program'?  'command' is more well used no?